actionpack-xml_parser.gemspec

```diff
@@ -1,10 +1,10 @@
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.name        = 'actionpack-xml_parser'
-  s.version     = '1.0.2'
+  s.version     = '2.0.0'
   s.summary     = 'XML parameters parser for Action Pack (removed from core in Rails 4.0)'
 
-  s.required_ruby_version = '>= 1.9.3'
+  s.required_ruby_version = '>= 2.2.1'
```

Comment:
2.2.1 passes though. I know Rails is set to 2.2.2 but I didn't want to remove 2.2.1 since it does work.